### PR TITLE
feat(labware-library): Support more pipettes in custom labware test protocol

### LINKED ISSUE #4789: Labware creator: add gen2 pipettes as an option in the test protocol
As a user, I would like to be able to be able to use Gen2 pipettes when testing my custom labware.

## Acceptance Criteria
- [ ] Allow users to select 'gen2' pipettes when exporting their custom labware and test protocol

### PR DESCRIPTION
**Pull request by SyntaxColoring**
## overview

The Custom Labware Creator generates a Python protocol to test the exported labware, but it only provides a few pipettes to pick from. 

This PR adds the rest of the single-channels. As far as I know, there's no reason for the test protocol *not* to support these pipettes. It was just never prioritized.

(People occasionally reach out to Support about this. We just edit the generated protocol and send them the modified file. So, if the rest of the pipettes *were* deliberately excluded, uh...)

Closes #4789.

## changelog

* Added dropdown entries for GEN2 single-channels and the P50S GEN1.
* Replaced some code that dynamically generated human-readable pipette names from their API load names, and vice-versa, with a big dumb table. Generalizing the code to support the GEN2s would have made it unnecessarily hard to follow.

## review requests

* I've eyeballed the generated protocol and made sure it simulates with `opentrons_simulate` for a few pipette choices, but double-check my work.
* JS is not my forté, so if there's a more idiomatic way to do any of this, let me know.
* Are the new names in the dropdown acceptable? They're intended to match what you see in the Opentrons App, but I'm working from memory.

### CODE REVIEW

File: labware-library/src/labware-creator/labwareTestProtocol.js [hunk 1]
```diff
@@ -2,14 +2,39 @@
...
+  'P1000 Single GEN1': {
+    loadName: 'p1000_single',
+    tiprack: 'opentrons_96_tiprack_1000ul',
+  },
 }
 
-export const pipetteNameOptions: Options = Object.keys(tiprackForPipette).map(
-  pipetteName => ({ name: pipetteName.replace(/_/g, ' '), value: pipetteName })
+export const pipetteNameOptions: Options = Object.keys(pipettes).map(
+  pipetteName => ({ name: pipetteName, value: pipetteName })
```
Comment (IanLondon) [inline labware-library/src/labware-creator/labwareTestProtocol.js]: I think this `value` should be loadName. These changes might be cleaner if `pipettes` is indexed by the loadName value and has `displayName` as a key, display names are UI and IMO ID's should be non-UI data

**Comment by SyntaxColoring** [comment]
@IanLondon 

# Changed

* `pipettes` is now indexed by load name.
* Dropdown `value`s are now the load name.
* Snuck in an update from `opentrons_96_tiprack_10ul` to `opentrons_96_tiprack_20ul`.

# Not changed

I attempted to replace all instances of `pipetteName` with `pipetteLoadName`, but:

* It shows up in a surprising number of places.
* I wasn't sure if it was appropriate to call it `pipetteLoadName` as opposed to a more opaque `pipetteID` or something.

# Also?

I also found this:

https://github.com/Opentrons/opentrons/blob/ca8ef0b621200beef07b070090ab9a960b3c25b8/labware-library/src/labware-creator/__tests__/loadAndSaveIntegration.test.js#L11-L12

I would have expected this to cause `make test-js` to fail, but it doesn't. I've updated it to say `p10_single` (lowercase), anyway.
